Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.10-bullseye
+FROM python:slim
```

Comment:
so the slim is okay now?

Reply:
Yes, this is already merged in separate code. Updated.